src/platform/graphics/shader-chunks/frag/webgpu.js

```diff
@@ -4,7 +4,22 @@ export default /* glsl */`
 #extension GL_EXT_samplerless_texture_functions : require
 
 layout(location = 0) out highp vec4 pc_fragColor;
+layout(location = 1) out highp vec4 pc_fragColor1;
+layout(location = 2) out highp vec4 pc_fragColor2;
+layout(location = 3) out highp vec4 pc_fragColor3;
+layout(location = 4) out highp vec4 pc_fragColor4;
+layout(location = 5) out highp vec4 pc_fragColor5;
+layout(location = 6) out highp vec4 pc_fragColor6;
+layout(location = 7) out highp vec4 pc_fragColor7;
+
 #define gl_FragColor pc_fragColor
+#define gl_FragColor1 pc_fragColor1
```

Comment:
I would suggest calling this render target instead of fragColor. Makes it more consistent with what we call the API on the CPU side.

Reply:
So, pc_renderTarget, pc_renderTarget1, etc.

Reply:
I would not want to call:
- output 0: **gl_FragColor** 
- output 1: pc_renderTarget1

especially since all of those are the same render target. Just different color buffers.

I prefer consistent:
```
gl_FragColor
gl_FragColor1
...
```

Reply:
Wouldn't it make more sense to get away from using gl_ prefixes and use our own, should make it easier to translate between glsl and wgsl for example.

Reply:
And this PR adds support for multiple render targets. How come it makes sense to have multiple render targets yet not refer to them as render targets?

Just not sure about the naming.

Reply:
We could move away, but we don't really want to rename existing gl_FragColor, as lots of shaders use it.
And naming 0th differently than following is not a great option either.
We could perhaps keep the old name and define a set of new names?

Reply:
Lets discuss in the afternoon.

Reply:
renamed to pcFragColor0 .. 7